scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts

```diff
@@ -126,10 +126,9 @@ export function ContentProfileSchemaEditor(vocabularies) {
 
             scope.getEditor3FormattingOptions = (fieldName) => {
                 const isCustomPlainTextField = typeof scope.fields[fieldName] === 'object'
-                    && typeof scope.fields[fieldName].field_options === 'object'
-                    && scope.fields[fieldName].field_options.single === true;
```

Comment:
What does `field_options.single` mean?

Reply:
Setting `field_options.single` to `true` means that the field is a single-line field. eg: slugline is a single-line field.